### ISSUE TITLE
ui settings launch issue

### ISSUE THREAD

**Issue by Deltoramastr** [Lailloken/Lailloken-UI#223]
so when i launch the ui it gives me a box that doesn't have the full list of options just general and wiki  clicking on those opens  the webpage help side but doesnt really show anything, ive tried uninstalling and re installing the ui and ahk any advice??
![Screenshot (12)](https://user-images.githubusercontent.com/26695758/230668585-6ccaea9d-6f0d-4179-bb56-f620dd454d71.png)

**Comment by Lailloken** [comment]
Are you running some kind of tool that removes the window border, or am I mistaken?

**Comment by Deltoramastr** [comment]
umm i think im in full screen mode but dont have the mouse confined to window do i need to have that set up before i can use this?

**Comment by Lailloken** [comment]
Nvm, it just looked like it because there was something near the edges of the client. Does it show any kind of warnings with regards to unsupported resolutions when you start the script?

**Comment by Deltoramastr** [comment]
yeah it has the unsupported resolution i adjusted it and it still has the same message (running 1280 x1024 now)

**Comment by Deltoramastr** [comment]
i was able to do the 1680 x900 but the window looks weird doing it this way. it wouldn't work with the 1024 for some reason

**Comment by Lailloken** [comment]
Yeah, 1024 is not supported any longer (among other resolutions) because it caused problems in the past with screen-reading. Affected users have to use a custom vertical resolution.

**Comment by Deltoramastr** [comment]
ohhh okay that makes more sense thanks :)

**Comment by Deltoramastr** [comment]
sorry but how do i minimize the box now that i have it open and working?

**Comment by Lailloken** [comment]
Doesn't it close if you click the [x]?
